### PR TITLE
Overhaul Using the list module

### PR DESCRIPTION
**Pull request by sarahmccarthy123**
Releases:main, Backport 12.4

### CODE REVIEW

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-create and manage records that don't have a dedicated module.
+The :guilabel:`Web > List` module allows you to browse through pages and folders
+in your site and view the records that
+are stored there. You can also use the :guilabel:`List` module to
+create and manage records without their own module.
 
-Sub pages, content elements and news stories are examples of the different
-kinds of records that can be stored on any given page.
+Records stored on pages include sub pages, content elements and news
+stories.
```
Comment (linawolf) [inline Documentation/ListModule/UsingEffectively/Index.rst:14]:
```suggestion
Records stored on pages include sub pages, content elements and news
stories.
```
"News stories" sounds strange here, I think usually we call them "News records". Should we mention that it depends on the setup of the system (installed use rights) and your user rights what you can actually see here?

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 2]
```diff
@@ -23,56 +23,56 @@ Select the :guilabel:`Web > List` module and browse to the
...
+8. **Copy URL** of this view
 
 9. Access **contextual help** (removed with version 12.0)
 
 .. todo: update screenshot
 
-In the **Content Area**, action icons are also available:
+Action icons are also available in the **Content Area**:
 
 .. include:: /Images/AutomaticScreenshots/ListModule/ListModuleContentArea.rst.txt
```
Comment (linawolf) [inline Documentation/ListModule/UsingEffectively/Index.rst:54]: Could you provide an updated screenshot showing TYPO3 v12 maybe?

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 3]
```diff
@@ -23,56 +23,56 @@ Select the :guilabel:`Web > List` module and browse to the
...
-   additional action icons (typically for accessing information, moving around, etc.)
+   additional action icons (typically used for accessing information, moving around, etc.)
 
-   Lastly, the cut and paste icons.
+   Lastly, there are cut and paste icons.
 
 .. todo: the "Extended view" option does not exist anymore
 
 If you wish to have the list of icons expanded all the time, you can check the
 :guilabel:`Extended view` box at the very bottom of the view.
```
Comment (linawolf) [inline Documentation/ListModule/UsingEffectively/Index.rst:72]: This option does not exist anymore. Remove it?

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 4]
```diff
@@ -23,56 +23,56 @@ Select the :guilabel:`Web > List` module and browse to the
...
 
-Also make sure that the :guilabel:`Show clipboard` box is checked, as we
+Now, make sure that the :guilabel:`Show clipboard` box is checked, as we
 will explore this next.
 
-If you plan to manage a multilingual web site, you will want to
-check the :guilabel:`Localization view` checkbox.
+If you plan to manage a multilingual web site, check the
```
Comment (linawolf) [inline Documentation/ListModule/UsingEffectively/Index.rst:77]: This option also does not exist anymore in TYPO3 v12 and even 11?

**Review by linawolf — COMMENTED** [review]
Thanks for your work on this, just some remarks

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 5]
```diff
@@ -5,13 +5,13 @@
...
 ============
 
-The :guilabel:`Web > List` module allows you to browse through
-each page or folder within your site and view all of the records that
-are stored within it. The :guilabel:`List` module also gives you the ability to
-create and manage records that don't have a dedicated module.
+The :guilabel:`Web > List` module allows you to browse through pages and folders
+in your site and view the records that
+are stored there. You can also use the :guilabel:`List` module to
+create and manage records without their own module.
```
Comment (brotkrueml) [inline Documentation/ListModule/UsingEffectively/Index.rst:11]: "without their own module" has another meaning than the previous "that don't have a dedicated module."

The new phrase assumes, there is an own module for a record (always), but you can use the List module instead. But this is the exception.

Comment (linawolf) [reply Documentation/ListModule/UsingEffectively/Index.rst:11]: @brotkrueml but even if there is a module for a record you can still use the list module?

File: Documentation/ListModule/UsingEffectively/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/ListModule/UsingEffectively/Index.rst:54]: I will take care of updating the screenshots in this manual.

File: Documentation/ListModule/UsingEffectively/Index.rst (hunk 5 above)
Comment (linawolf) [reply Documentation/ListModule/UsingEffectively/Index.rst:11]: To me it reads: If there was not list module you would need a dedicated module for each record

File: Documentation/ListModule/UsingEffectively/Index.rst (hunk 2 above)
Comment (sarahmccarthy123) [reply Documentation/ListModule/UsingEffectively/Index.rst:54]: I'll just do this one. I'm trying to learn Gimp :-)

Comment (brotkrueml) [reply Documentation/ListModule/UsingEffectively/Index.rst:53]: Tried to draw an arrow with Gimp. Had to install a plugin (how do I install a plugin?) After that: Where it that plugin, and then it hasn't worked. Switched to Photoshop Elements -> done in 3 minutes (including starting the Windows VM).

Good luck.

**Review by brotkrueml — COMMENTED** [review]
The numbers in the rectangles look strange. Also I suggest to set all image rectangles with the numbers to a common baseline.

**Review by brotkrueml — COMMENTED** [review]
Hmm, don't know what you are doing, but the numbers/texts are still pixelated.

**Comment by linawolf** [comment]
@brotkrueml can we deal with the pixelated screenshots in a follow up? The whole topic of the best way to alter screenshots with numbers etc is unresolved anyway. Other then that the screenshots are clear and readable and the changes worthwhile merging?

**Review by linawolf — APPROVED** [review]

**Comment by brotkrueml** [comment]
@linawolf Then leave the screenshot as is for now. But feel free to merge AND create an issue about is so this is not forgotten.

**Comment by linawolf** [comment]
@sarahmccarthy123 Thanks for improving this chapter!

**Pull request merged**